spec/omniauth/strategies/ethereum_spec.rb

```diff
@@ -0,0 +1,96 @@
+RSpec.describe OmniAuth::Strategies::Ethereum do
```

Comment:
I'm getting:

```
NameError:
  uninitialized constant OmniAuth
# ./spec/omniauth/strategies/ethereum_spec.rb:1:in `<top (required)>'
```

Can you rebase on latest main, so we can make sure the `rspec` job passes on neutral grounds?

Reply:
Rebase completed.